recipes/cubicinterpolation/all/test_v1_package/test_package.cpp

```diff
@@ -0,0 +1,22 @@
+#include "CubicInterpolation/Axis.h"
+#include "CubicInterpolation/CubicSplines.h"
+#include "CubicInterpolation/Interpolant.h"
+
+double func(double x) { return x * x + x; }
+
+int main(int argc, char* argv[])
```

Comment:
You can remove this duplciated file if you update the CMakelsits.txt with the new template version 
https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_v1_package/CMakeLists.txt

Reply:
This file is no longer used 🚀 you can delete it please